### DISCUSSION TITLE
curl -L \   -H "Accept: application/vnd.github+json" \   -H "Authorization: Bearer <YOUR-TOKEN>" \   -H "X-GitHub-Api-Version: 2022-11-28" \   https://api.github.com/repos/OWNER/REPO/hooks/HOOK_ID

### DISCUSSION THREAD

**Discussion by Kaliifmumegilu95** [community/community#113932] [New to GitHub]
curl -L \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/OWNER/REPO/hooks/HOOK_ID

**Comment by devashish2024** [comment]
What are you trying to do? Consider asking me what you need help with.

You need to pass a bearer token for authorization and specify the route properly
